src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -36,8 +36,8 @@ public class GenevaLogExporter : GenevaBaseExporter<LogRecord>
     private readonly string m_defaultEventName = "Log";
     private readonly IReadOnlyDictionary<string, object> m_prepopulatedFields;
     private readonly List<string> m_prepopulatedFieldKeys;
-    private static readonly ThreadLocal<byte[]> m_buffer = new ThreadLocal<byte[]>(() => null);
-    private readonly byte[] m_bufferEpilogue;
+    private static readonly ThreadLocal<byte[]> m_buffer = new ThreadLocal<byte[]>();
+    private static readonly byte[] m_bufferEpilogue = new byte[25]; // This size was determined by precomputing the number of bytes required to serialize the epilogue.
```

Comment:
Given it's now static instead of member variable:

```suggestion
    private static readonly byte[] s_bufferEpilogue = new byte[25]; // This size was determined by precomputing the number of bytes required to serialize the epilogue.
```